### ISSUE TITLE
The urls for lots of emojis are not existing anymore.

### ISSUE THREAD

**Issue by chensivan** [JoshCheek/bitmoji#3]

**Comment by MichaelQuint** [comment]
Yes i'm running into the same issue, whats even more concerning is that i've only been using there API for a week and i'm seeing huge inconsistencies with the servers. I may build an app that will store them in a cache directory or something that way it doesn't impact my system if they decide to remove older expressions.

**Comment by MichaelQuint** [comment]
It looks like they are deprecating the scale parameter?
See example:
(Works as of 1/25/2018)
https://render.bitstrips.com/v2/cpanel/9090071-396003945_22-s1-v1.png?transparent=1&scale=1

(Does not work as of 1/25/2018, but was working up till 1/22/2018)
https://render.bitstrips.com/v2/cpanel/9090071-396003945_22-s1-v1.png?transparent=1&scale=4

**Comment by skoshy** [comment]
Both seem to be working for me @MichaelQuint

**Comment by bmsimo** [comment]
November 25, 2018 ;
Scale=4 doesn't work but scale=2 does

**Comment by MichaelQuint** [comment]
I confirm scale=4 is no longer working, last time this happend the images came back up a few days later. Scale=2 works OK, but are not as high res as scale=4.
